frontend/src/routes/(authenticated)/project/[project_code]/ResetProjectModal.svelte

```diff
@@ -89,6 +89,7 @@
     try {
       const url = `/api/project/finishResetProject/${code}`;
       const resetResponse = await fetch(url, { method: 'post' });
+      console.log(new Date(), resetResponse.ok ? 'finishResetProject succeeded' : 'finishResetProject failed')
```

Comment:
debugging left behind?

Reply:
Removed in commit a581c380ccf4b638e1080861bdcd48a654349b98.